R/plot_jags_densitydx.R

```diff
@@ -0,0 +1,110 @@
+
+#' @title Density Plot Diagnostics
+#' @author Sam Schildhauer
+#' @description
+#'  plot_jags_dens() takes a [list] output from [serodynamics::run_mod()]
+#'  to create density plots for each chain run in the mcmc estimation.
+#'  Defaults will produce every combination of antigen/antibody, parameters,
+#'  and stratifications, unless otherwise specified.
+#'  Antigen/antibody combinations and stratifications will vary by analysis.
+#'  The antibody dynamic curve includes the following parameters:
+#'  - y0 = baseline antibody concentration
+#'  - y1 = peak antibody concentration
+#'  - t1 = time to peak
+#'  - r = shape parameter
+#'  - alpha = decay rate
+#' @param data A [base::list()] outputted from run_mod().
```

Comment:
```suggestion
#' @param data A [list] outputted from run_mod().
```

Crosslinks to functions and classes from the 6 default R packages ("datasets"  "utils"     "grDevices" "graphics"  "stats"     "methods") don't need to be explicitly prefixed with their package name; `roxygen2` will figure it out for you.

Also, when you're referring to a function, I think it's best to include the parentheses in the crosslink, but when you're referring to an object class, like you're doing here, I think it's best to omit the parentheses.